### PR TITLE
fix(UX-922): FAB-redesign

### PR DESCRIPTION
**Pull request by DE7924**
docs: added expanded knob to widgetbook

fix: added label to non-expanded fab

### CODE REVIEW

File: lib/src/components/fabs/fab.dart [hunk 1]
```diff
@@ -113,58 +113,64 @@ class _ZetaFABState extends State<ZetaFAB> {
...
+                    )
+                  : EdgeInsets.all(widget.size.padding(context)),
+              child: Row(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  ZetaIcon(widget.icon, size: widget.size.iconSize(context)),
+                  if (widget.expanded && widget.label != null)
+                    Row(
+                      mainAxisSize: MainAxisSize.min,
+                      children: [Text(widget.label!, style: ZetaTextStyles.labelLarge)],
```
Comment (mikecoomber) [inline lib/src/components/fabs/fab.dart:165]: This text should probably wrap if it gets too long, we don't want it too expand too far past the width of the button itself

File: lib/src/components/fabs/fab.dart [hunk 2]
```diff
@@ -113,58 +113,64 @@ class _ZetaFABState extends State<ZetaFAB> {
...
+              },
+            ),
+          ),
+          child: AnimatedContainer(
+            duration: ZetaAnimationLength.normal,
+            child: Padding(
+              padding: widget.expanded
+                  ? EdgeInsets.symmetric(
+                      horizontal: Zeta.of(context).spacing.large,
+                      vertical: Zeta.of(context).spacing.medium,
```
Comment (mikecoomber) [inline lib/src/components/fabs/fab.dart:155]: You might need to double check the spacing on this, it looks too small when compared to the designs
image
image

File: lib/src/components/fabs/fab.dart (hunk 1 above)
Comment (DE7924) [reply lib/src/components/fabs/fab.dart]: I've added a constraint of 100. I was looking for a decent way to set this dynamically but I fell short. Any thoughts?

File: lib/src/components/fabs/fab.dart (hunk 2 above)
Comment (DE7924) [reply lib/src/components/fabs/fab.dart:155]: I've increased the spacing 👍

**Review by mikecoomber — APPROVED** [review]

**Pull request merged**